### PR TITLE
Update actions to deploy all releases to quay

### PR DESCRIPTION
**Pull request by metalice**
Signed-off-by: Matan Schatzman <mschatzm@redhat.com>

## 📝 Description

Currently deploy to quay don't support other branches, only main. this should be fix that every merge to other branches will release a new image to quay
## 🎥 Demo

> Please add a video or an image of the behavior/changes

### CODE REVIEW

**Review by upalatucci — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -26,12 +28,11 @@ jobs:
...
-          push: true
-          tags: quay.io/kubevirt-ui/kubevirt-plugin:latest
+      # - name: Build and push
+      #   id: docker_build
+      #   uses: docker/build-push-action@v2
+      #   env:
+      #     GITHUB_TOKEN: "${{ secrets.BOT_PAT }}"
+      #   with:
+      #     push: true
+      #     tags: quay.io/kubevirt-ui/kubevirt-plugin:$VERSION_TAG
```
Comment (avivtur) [inline .github/workflows/deploy.yml:38]: ?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -26,12 +28,11 @@ jobs:
...
-
-      - name: Build and push
-        id: docker_build
-        uses: docker/build-push-action@v2
-        env:
-          GITHUB_TOKEN: "${{ secrets.BOT_PAT }}"
-        with:
-          push: true
-          tags: quay.io/kubevirt-ui/kubevirt-plugin:latest
+      # - name: Build and push
```
Comment (pcbailey) [inline .github/workflows/deploy.yml:31]: Is there a reason this code is commented out?

**Pull request closed without merging**